src/sections/Community/Handbook/connect.js

```diff
@@ -16,37 +23,96 @@ const Connect= () => {
+            <p><a href="https://layer5io.slack.com/archives/C0130BQ4L3T">#mesheryctl</a> - For discussions on Meshery’s CLI</p>
+            <p><a href="https://layer5io.slack.com/archives/C019426UBNY">#newcomers</a> - For welcoming first-time contributors and community newcomers.</p>
+            <p><a href="https://layer5io.slack.com/archives/CVBGPTFJN">#performance</a> - For discussions related to measuring and evaluating Service Mesh Performance.</p>
+            <p><a href="https://layer5io.slack.com/archives/C010H0HE2E6">#smi</a> - For discussions related to Meshery and Service Mesh Interface (SMI).</p>
+            <p><a href="https://layer5io.slack.com/archives/C012UMS2MCM">#wasm</a> - For discussions on web assembly, rust, data plane filters.</p>
+            <p><a href="https://layer5io.slack.com/archives/C015QJKUMPU">#websites</a> - For discussions related to the UI/UX of <a href="http://layer5.io/">Layer 5</a> project sites.</p>
+
+            <h2>Discourse Forum</h2>
+            <p>Join us on <a href="https://discuss.layer5.io">Discourse</a></p>
+
+            <h2>Social Media</h2>
+            <p>Layer5 is dedicated to providing the most efficient Service mesh management tool while growing a healthy open-source community. To know more about layer5 and connect with our community, follow and engage with us and join our social media accounts.</p>
+            <ul>
+              <li>
+                <img height="40 px" src={twitter_icon} alt="twitter" />
+                Twitter: 
+                <a href="https://twitter.com/layer5">@layer5, @mesheryio & @smp_spec</a>
+              </li>
+              <li>
+                <img height="40 px" src={mail_icon} alt="mail" />
+                Mail: 
+                <a href="mailto:community@layer5.io">community@layer5.io</a>
+              </li>
+              <li>
+                <img height="40 px" src={youtube_icon} alt="youtube" />
+                Youtube: 
+                <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA">Subscribe on Youtube</a>
+              </li>
+              <li>
+                <img height="40 px" src={github_icon} alt="github" />
+                GitHub: 
+                <a href="https://github.com/layer5io">Layer5io</a>
+              </li>
+              <li>
+                <img height="40 px" src={linkedin_icon} alt="linkedin" />
+                LinkedIn: 
+                <a href="https://www.linkedin.com/company/layer5">Layer 5</a>
+              </li>
+              <li>
+                <img height="40 px" src={docker_icon} alt="docker" />
+                Docker: 
+                <a href="https://hub.docker.com/u/layer5/">Layer 5</a>
+              </li>
+              <li>
+                <img height="40 px" src={slack_icon} alt="slack" />
+                Slack: 
+                <a href="https://bit.ly/3fXwYPk">Layer 5</a>
+              </li>
+            </ul>
```

Comment:
![icons](https://user-images.githubusercontent.com/85789734/132383753-ed9d069d-4ca6-4162-b4cf-70eeeb0783d0.jpg)
1) Please make these social icons a bit smaller and of same size.
2) Increase the space between the icons and the text.
3) Remove the bullet points here.
4) Remove the platform names if you're adding the icons.

Reply:
Done!!